src/Libraries/UnitsUI/Properties/Resources.en-US.resx

```diff
@@ -150,6 +150,27 @@
     <value>units;unittypes;</value>
     <comment>Search tags. When do translation, appending localized tags to the existing tags and use ';' to seperate them</comment>
   </data>
+  <data name="UnitValueOutputDescription" xml:space="preserve">
+    <value>A custom formatted display of a unit value.</value>
+    <comment>Temporary description fro 'unit value output node'</comment>
```

Comment:
not sure i understand this comment?

Reply:
I guess we generally want to remove comments? I thought we may want placeholder comments. Guess not.

Reply:
Ah nvm, we do want a placeholder.  Will follow what the other nodes are doing.